go.mod

```diff
@@ -192,3 +192,5 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 	tags.cncf.io/container-device-interface v0.8.0 // indirect
 )
+
+replace github.com/compose-spec/compose-go/v2 => github.com/glours/compose-go/v2 v2.0.0-20240924124937-38f94a79402c
```

Comment:
?

Reply:
Just to test again Compose CI until we release a new version of compose-go

Reply:
I marked the PR as draft for now

Reply:
right, I got confused for this reason as I got notified on this one first :)